R/tm_a_regression.R

```diff
@@ -843,6 +856,9 @@ srv_a_regression <- function(id,
     )
 
     output$text <- renderText({
+      req(iv_r()$is_valid())
```

Comment:
so the way this is done, you now don't get all the things that are wrong shown:
![image](https://user-images.githubusercontent.com/15201933/209347294-58cd2e99-b1fb-4bff-84a8-91cf96c574d8.png)

and I'd argue the theme choice is much less important than you haven't selected a variable